.github/workflows/welcome-newcomers.yaml

```diff
@@ -0,0 +1,24 @@
+steps:
+  - uses: actions/first-interaction@v1
+    with:
+      repo-token: ${{ secrets.GITHUB_TOKEN }}
+      issue-message: |
+        Welcome to the Pantsbuild Community. This looks like your first issue here.
+
+        If you haven't already, come say hi on Slack.
+        Use [this link](https://docs.google.com/forms/d/e/1FAIpQLSf9zgf-MXRnVDJbrVEST3urqneq7LCcy0zw8qU-GH4hPMn52A/viewform?usp=sf_link)
+        for an invitation.
+
+        Thanks again, and we look forward to your next Issue/PR :smile:!
+
+      pr-message: |
+        Welcome to the Pantsbuild Community. This looks like your first PR here.
+
+        We are so grateful for each an every contribution, big or small.
+        So thanks for spending your time and energy making this project better.
```

Comment:
Really nice.

Reply:
Come for the tech, stay for the community.